e3sm_diags/derivations/acme.py

```diff
@@ -705,6 +725,10 @@ def cosp_histogram_standardize(cld: "FileVariable"):
                 lambda fsntoa, fsntoac: swcf(fsntoa, fsntoac),
             ),
             (("rsut", "rsutcs"), lambda rsutcs, rsut: swcf(rsut, rsutcs)),
+            (
+                ("SW_flux_up_at_model_top", "SW_clrsky_flux_up_at_model_top"),
+                lambda rsutcs, rsut: swcf(rsut, rsutcs),
```

Comment:
The final "SWCF" plot looks correct, so I guess it's not an issue. 

https://portal.nersc.gov/cfs/e3sm/kzhang/e3sm_diag/lat_lon_eamxx_09/lat_lon/CERES-EBAF-TOA-v4.1/ceres_ebaf_toa_v4.1-SWCF-09-global.png

Reply:
Hi @kaizhangpnl thanks for looking into it! I was also supersized that the final plot looks reasonable, still investigating...

Reply:
Going through the code for derived variable and documentation for lamda expression. It does seem like the order of the parameters **after** `lambda` does not matter. 

In the derived variable dictionary, it seems like we can further simplify without using lamda expression, just to use the regular functions for most if not all cases. Tagging @tomvothecoder for confirmation..

Reply:
> Going through the code for derived variable and documentation for lamda expression. It does seem like the order of the parameters **after** `lambda` does not matter.
> 
> In the derived variable dictionary, it seems like we can further simplify without using lamda expression, we can just to use the regular functions for most if not all cases. Tagging @tomvothecoder for confirmation..

That's correct, the lambda isn't needed because the required args will be passed directly to the function when it is called. The lambda is doing the same thing here.

Reply:
Thank you @tomvothecoder !